data/arweave/assets/l4iqeiSb4oJrpByg6rgiXlW1iF3cgjXLbHdG2JvAC_c/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "fQR weave",
+  "symbol": "FQR",
+  "web": "https://fqrweave.tech",
+  "supply": 100000000,
+  "whitepaper": "https://fqrweave.tech/docs/lightpaper.pdf",
+  "product": true,

```

Comment:
your product does not have real-usage onchain with a minimum of users, remove it